server/shared/src/test/scala/org/http4s/server/middleware/ResponseTimingSuite.scala

```diff
@@ -45,7 +46,7 @@ class ResponseTimingSuite extends Http4sSuite {
 
     val header = res
       .map(_.headers.headers.find(_.name == ci"X-Response-Time"))
-    header.map(_.map(_.value.toInt) === Some(artificialDelay)).assert
+    TestControl.executeEmbed(header.map(_.map(_.value.toInt) === Some(artificialDelay))).assert
```

Comment:
I can't comment on it directly because it's above your changes 👆 but what is this `Sys.tick()` stuff? Where is it coming from. 

Aha, it's down below 👇 I think we can toss that. We don't need it now that we have `TestControl` 😂

Reply:
Yep, I also found out about [this guy](https://github.com/http4s/http4s/blob/series/0.23/client/shared/src/test/scala/org/http4s/client/middleware/RetrySuite.scala), and I'm going to use `TestControl` there too - hold on tight ;-)

Reply:
While I'm fixing this, I thought: shouldn't we put in some contributions guidelines that it would be recommended to use `TestControl` anywhere in tests the passing of time is a factor? If so, I'd happily make a PR to update the documentation and explain the pattern

Reply:
Yep, that would be great!

Reply:
Would you mind pointing me to where the contributing guidelines sit, so that I can make the change? :-)

Reply:
https://github.com/http4s/http4s/blob/c63372cc886bef753a62a90009e9dd8df8330a51/docs/contributing.md

Reply:
done